metals/src/main/scala/scalafix/languageserver/ScalafixEnrichments.scala

```diff
@@ -40,9 +40,10 @@ object ScalafixEnrichments {
   implicit class XtensionPatchLSPObject(val `_`: Patch.type) extends AnyVal {
     def lintMessagesInternal(
         patches: Map[RuleName, Patch],
-        ctx: RuleCtx
+        ctx: RuleCtx,
+        index: SemanticdbIndex
     ): List[LintMessage] =
-      Patch.lintMessages(patches, ctx)
+      Patch(patches, ctx, Some(index))._2
```

Comment:
Minor: for readability, I tend to shy away from `_N` methods in tuples.
Can we do:

```scala
val (_, lintMessage) = Patch(patches, ctx, Some(index))
lintMessage
```

instead?

Reply:
Done.